authz/src/main/java/io/grpc/authz/AuthorizationPolicyTranslator.java

```diff
@@ -44,7 +45,8 @@
 
   private static StringMatcher getStringMatcher(String value) {
     if (value.equals("*")) {
-      return StringMatcher.newBuilder().setPrefix("").build();
+      return StringMatcher.newBuilder().setSafeRegex(
+        RegexMatcher.newBuilder().setRegex("^\\S+$").build()).build();
```

Comment:
Shouldn't this be `.+`? Seems like this is over-specifying and would match incorrectly (e.g., for headers).

Reply:
Done